strings/is_isogram.py

```diff
@@ -0,0 +1,32 @@
+"""
+wiki: https://en.wikipedia.org/wiki/Heterogram_(literature)#Isograms
+"""
+
+
+def is_isogram(string: str) -> bool:
+    """
+    An isogram is a word in which no letter is repeated.
+    Examples of isograms are uncopyrightable and ambidextrously.
+    >>> is_isogram('Uncopyrightable')
+    True
+    >>> is_isogram('allowance')
+    False
+    >>> is_isogram('copy1')
+    Traceback (most recent call last):
+     ...
+    ValueError: String must only contain alphabetic characters.
+    """
+    import re
+
+    if bool(re.search(r"\d", string)):
```

Comment:
```suggestion
    if not all(x.isalpha() for x in string):
```
This reduces the need to import `re` and catches things other non alphabetic characters such as symbols.